matter_server/server/server.py

```diff
@@ -85,6 +85,7 @@ async def start(self) -> None:
                 "CHIP Core version does not match CHIP Clusters version."
             )
         self.loop = asyncio.get_running_loop()
+        await self.device_controller.initialize()
```

Comment:
The issue is that `compressed_fabric_id` isn't set until the device controller is initialized, and we need the fabric id for loading from the storage path, right?

Reply:
yep